.github/workflows/macOS.yml

```diff
@@ -21,7 +21,7 @@ jobs:
 
       - name: build project
         run: |
-          make package DEBUG=0
+          make package ENABLE_VALIDATION_LAYERS=0
```

Comment:
why does this need to be explicitly defined?

Reply:
ah, you're right. I think I just saw it and replaced the variables.

Reply:
I'll remove during the rebase.